e2e/community_detection_online_test/test_set/test.yml

```diff
@@ -1,5 +1,5 @@
 query: >
-  CALL dynamic_community_detection.get()
+  CALL community_detection_online.set()
```

Comment:
Should be .get()

Reply:
Did you think "set" since that's the name of the directory?